### DISCUSSION TITLE
no connection at all since update 1.3.4

### DISCUSSION THREAD

**Discussion by Rapha0815** [Chia-Network/chia-blockchain#11377] [Support]
Hi guys,

after I installed the client version 1.3.4 (using an Intel iMac 2017 5K, running on macOS 12.3.1), I had regular trouble with synchronisation. Sometimes it worked, then for many hours didn't.

So I restarted the app – and since then I see the white screen with "Starting services – full node" only. Restarted the Mac, no change, still the same. Since almost 3 days now only white screen. I don't even reach the point, where I choose the key.

What can I do that it works again? Is maybe a 1.3.5 on the way, which fixes this already?

I never had similar trouble until 1.3.3, I'm farming since the main net started one year ago.

<img width="279" alt="srcc" src="https://user-images.githubusercontent.com/104693965/166119126-a420877f-f023-4e65-bf1d-09a90f0b3bd6.png">

**Comment by loppefaaret** [comment]
If it is stuck on the "Full Node" part of the startup, something is preventing the Full Node process from initializing correctly
Could you try and locate your `debug.log` located under your users home folder, in `/Users/<USERNAME>/.chia/mainnet/log/` 
Once in your users home folder, you need to switch on showing hidden folders from the top bar drop down menu, to see the `.chia/` folder
If you then could attach the `debug.log` file to this topic, i would be able to try and figure out why exactly it is having problems starting up the full node process

**Reply by Rapha0815** [reply]
no idea?

**Comment by Rapha0815** [comment]
Thank you very much for your answer

[debug.log](https://github.com/Chia-Network/chia-blockchain/files/8619608/debug.log)

**Reply by loppefaaret** [reply]
First, i want to ensure that you got your database on a fast drive, and not on an HDD - the perfomance on almost any HDD is suboptimal.
to help me better diagnose your problem on macOS, first reboot the machine, and do not start Chia, and then you would have to open the "Terminal" app, and execute the below commands on line at a time:
```
cd /Applications/Chia.app/Contents/Resources/app.asar.unpacked/daemon
./chia configure --log-level INFO
mv ~/.chia/mainnet/debug.log ~/.chia/mainnet/log/debug.log.old
./chia start node
```
if you see any reported errors in the Terminal window after executing the above commands, please copy and paste them here, and after letting it run for a minute or two, please attach the new `debug.log` file created after this single run.
